app/views/devise/registrations/edit.html.erb

```diff
@@ -1,53 +0,0 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
I think we need this view for user edit functionality. User should be able to edit his profile upto some extent like password, name etc(Not the email, roles or any other critical stuff)

Reply:
I think we have other route for this